### DISCUSSION TITLE
Removing favorited comments

### DISCUSSION THREAD

**Discussion by th0b3n** [Razeeman/Android-SimpleTimeTracker#247] [Q&A]
Is there a way to remove favorited comments without adding an activity?

**Comment by Razeeman** [comment]
Hello! Not sure that is meant by "without adding an activity" but favorited comment can be removed by opening edit of any record or clicking "+" button on the second tab, and then going to Comment section, selecting favourite comment and clicking on Star button to remove it from favourites. Is this what you are looking for?
